packages/flutter/lib/src/rendering/object.dart

```diff
@@ -1319,6 +1340,8 @@ abstract class RenderObject extends AbstractNode implements HitTestTarget {
     return _layer;
   }
 
+  /// Whether this render object requires an update to its compositing state.
+  bool get needsCompositingBitsUpdate => _needsCompositingBitsUpdate;
```

Comment:
It looks like you're exposing this bit just for testing.  The problem with exposing these bits is that people will use them to write wrong code.  Is there a way of testing whether the `attach` code works correctly without reading these bits?  For example, presumably if we fail to dirty things properly we'll compute the wrong layer tree in the end.